oauth2server/apps/accounts/views.py

```diff
@@ -23,8 +27,8 @@ def post(self, request, *args, **kwargs):
         :param kwargs:
         :return: Serialised JSON Response Object to indicate the resource has been created
         """
-        print "hitting post account view"
-        print "User object is: ", request.user
+        stdlogger.info( "Hitting post account view method")
+        stdlogger.debug( "User object is: ", request.user )
```

Comment:
TypeError: not all arguments converted during string formatting
Logged from file views.py, line 31